src/renderer/components/+custom-resources/crd-list.tsx

```diff
@@ -77,7 +77,7 @@ class NonInjectedCustomResourceDefinitions extends React.Component<Dependencies>
       ? "Group"
       : "Groups";
 
-    return `${prefix}: ${iter.join(this.selectedGroups, ", ")}`;
+    return `${prefix}: ${iter.join(this.selectedGroups.values(), ", ")}`;
```

Comment:
This requirement of iterable is artificial.

Also for comparison: 
`${prefix}: ${iter.join(this.selectedGroups.values(), ", ")}`
vs.
`${prefix}: ${this.selectedGroups.join(", ")}`

Reply:
Incorrect, `selectedGroups` is not an array.

Reply:
But it should be an array.

Reply:
Why? An array has the wrong semantics entirely, that is why it is a `Set<T>`. Arrays are for lists of items in a definite order, with possible duplicates. Those are not the semantics we want.